docs/reference/server-configuration.md

```diff
@@ -446,13 +446,25 @@ application. The stream processing code runs in these threads. For more
 information about the {{ site.kstreams }} threading model, see
 [Threading Model](https://docs.confluent.io/current/streams/architecture.html#threading-model).
 
-## ksql.streams.processing.guarantee
+## `ksql.streams.processing.guarantee`
 
 The processing semantics to use for persistent queries. The default is 
 `at_least_once`. To enable exactly-once semantics, use `exactly_once`. 
 
 For more information, see [Processing Guarantees](/operate-and-deploy/exactly-once-semantics).
 
+## `ksql.streams.producer.compression.type`
+
+The type of compression used by streams producers for topics created by 
+CREATE TABLE AS SELECT and CREATE STREAM AS SELECT statements.
+This setting is distinct from the `compression.type` config, which sets the
```

Comment:
Seem there is no entry for `ksql.compression.type` ? Should we add one? (also not the `ksql.` prefix)

Reply:
Thanks @mjsax, I didn't realize that's also a ksqlDB config. I added it further up the page (in alpha order).

Reply:
\cc @mikebin to confirm that this PR look ok :)